discord/colour.py

```diff
@@ -232,7 +245,26 @@ def blurple(cls):
 
     @classmethod
     def greyple(cls):
-        """A factory method that returns a :class:`Colour` with a value of ``0x99aab5``."""
+        """A factory method that returns a :class:`Colour` with a value of ``0x99aab5``.
+        .. versionchanged:: 1.5
+            Added grayple alias.
+        """
         return cls(0x99aab5)
 
+    grayple = greyple
+
+    @classmethod
+    def black(cls):
+        """A factory method that returns a :class:`Colour` with a value of ``0x000001``.
+        .. versionadded:: 1.5
+        """
+        return cls(1)
```

Comment:
Colours in the factory methods correspond to either the branding guidelines or the role colour picker, adding colours to work around Discord's weird reservation of a specific colour seems a bit off to me so I'll have to reject it.

If you want to have one added here then it could be `not_quite_black` which would correspond to `0x23272A` but I'm not sure I've seen anyone request this.

Reply:
Fairs. Thought I'd propose these since I was in the file anyways.

I'm happy to add the `not_quite_black` one in if you felt like it would fit. Having looked at the branding link you sent in an above comment, it does appear, so might as well I spose?